eclipse-collections-code-generator/src/main/resources/impl/list/mutable/unmodifiablePrimitiveList.stg

```diff
@@ -292,6 +294,12 @@ public class Unmodifiable<name>List
     {
         return this.getMutable<name>List().collectWithIndex(function, target);
     }
+
+    @Override
+    public Mutable<name>Stack toStack()
```

Comment:
This method should be pulled up to `AbstractUnmodifiable<name>Collection`.

Reply:
I have it here as AbstractUnmodifiable<name>Collection is not implementing Ordered<name>Iterable.